recipes/jupyterlab-nbconvert-nocode/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "jupyterlab_nbconvert_nocode" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a7c1617d2b49d425984d30e5e0cc9e7e1b354bb570dab474cafffb70f61b5162
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+outputs:
+  - name: jupyterlab_nbconvert_nocode
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python >=3.6
+        - pip
+        - jupyter-packaging
+      run:
+        - python >=3.6
+        - jupyterlab >=1.0.0
+        - nbconvert >=6.0.0
+      run_constrained:
+        - {{ pin_subpackage("jupyterlab-nbconvert-nocode", max_pin="x.x.x") }}
+
+    test:
+      imports:
+        - jupyterlab_nbconvert_nocode
+
+  - name: jupyterlab-nbconvert-nocode
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python >=3.6
+      run:
+        - python >=3.6
+        - {{ pin_subpackage("jupyterlab_nbconvert_nocode", max_pin="x.x.x") }}
```

```suggestion
      run:
        - {{ name|lower }} {{ version }}
```
By specifying the exact version you'll make sure that everything gets installed correctly, even if the max_pin is not applicable for some reason.